src/commcare_cloud/fab/operations/airflow.py

```diff
@@ -1,13 +1,20 @@
 from __future__ import absolute_import
 
+import os
+
 from fabric.api import roles, sudo, env
-from fabric.context_managers import cd
+from fabric.context_managers import cd, shell_env
 
 from ..const import ROLES_AIRFLOW
 
+
 @roles(ROLES_AIRFLOW)
 def update_airflow():
-    with cd(env.airflow_code_root):
+    deploy_key = 'deploy_key_airflow.pem'
```

Comment:
you need to update `environmetns/<env>/meta.yml` to list the new deploy key:

```diff
 deploy_keys:
   deploy_key.vault: deploy_key.pem
+  deploy_key_airflow.vault: deploy_key_airflow.pem
```

Reply:
added